src/OrbitDB.js

```diff
@@ -114,8 +142,12 @@ let databaseTypes = {
 
   async disconnect () {
     //close Keystore
-    if (this.keystore.close)
-      await this.keystore.close()
+    if (this.keystore._store.close)
+      await this.keystore._store.close()
```

Comment:
I think these (this and the one for cache below) would be nicer if they were just:
`await this.keystore.close()`

Not sure how to achieve that though...

Reply:
Couldnt `this._store.close()` be called in `keystore.close()`?

Reply:
That's correct - I'll re-implement that function inside `orbit-db-keystore` and `orbit-db-cache`. I was trying to avoid those modules controlling the storage at that level, but eh... 

Interacting directly vs interacting with "internal" variables at the `OrbitDb` level is six in one hand, half dozen in the other.

I'll implement `this.keystore.close`